core/sail/elasticsearch-store/src/test/java/org/eclipse/rdf4j/sail/elasticsearchstore/TestHelpers.java

```diff
@@ -22,7 +22,7 @@
 public class TestHelpers {
 
 	private final static Random random = new Random();
-	public static final String VERSION = "6.5.4";
+	public static final String VERSION = "6.8.7";
```

Comment:
Is this a change we need? From your comments I got the impression you merely tried to use this to fix some test failures - or were those _different_ test failures? 

If we don't need this for Java 13, let's roll it back so we can merge this independently of the Elasticsearch version bump.